src/components/MainPlayer.vue

```diff
@@ -17,6 +17,7 @@
         <a
           :href="titleLink"
           class="main-player-title-link"
+          target="_blank"
         >
```

Comment:
In this case these links will probably usually go to domains we control but we should add `rel=noopener` to links that can go offsite as a best practice. There are probably already linter rules available that we can set up for this eventually.

Reply:
good catch, I'll update - this will help our page speed score too